packages/api-core/src/graphql/schema.graphql

```diff
@@ -34,6 +34,130 @@ enum MassUnit {
   oz
 }
 
+"Logical Operator Types for filtering"
+enum LogOpTypes{
+  "AND"
```

Comment:
This could use a really well written description/explanation here. GraphQL interfaces should be self-documenting whenever possible.